extensions/msal-node-extensions/src/broker/NativeBrokerPlugin.ts

```diff
@@ -656,50 +656,73 @@ export class NativeBrokerPlugin implements INativeBrokerPlugin {
         ) {
             const { errorCode, errorStatus, errorContext, errorTag } =
                 error as MsalRuntimeError;
-            const tagString = StringUtils.tagToString(errorTag);
+            const tagString = NativeBrokerStringUtils.tagToString(errorTag);
             const enhancedErrorContext = errorContext
                 ? `${errorContext} (Error Code: ${errorCode}, Tag: ${tagString})`
                 : `(Error Code: ${errorCode}, Tag: ${tagString})`;
```

Comment:
Can we do this stuff in the `PlatformBrokerError` constructor instead? Then you don't need to export the Util function at all.